docs/rfc/rfc-2.adoc

```diff
@@ -0,0 +1,811 @@
+<<bitcoin-fee>>), construct the sighash (see <<sighash-construction>>) and
+proceed into the signing process to sign that sighash. Once a signature is
+computed, each client should construct the final transaction (see
+<<final-transaction-construction>>) and submit it (see
+<<transaction-submission>>).
+
+When receiving the ready message from other nodes, a node should validate that
+the announced BTC address can be parsed correctly by the tooling used to handle
+transaction and sighash construction. If the announced BTC address cannot be
+parsed, an error should be logged and the message should be treated as if it was
+not received.
+
+The retry mechanism for the signing process for liquidation recovery should be
+identical to regular signing, repeating every 2 minutes that a signature has
+not been completed successfully until the liquidation recovery timeout. Unlike
+standard signing, the liquidation recovery timeout should be very long to allow
+for node malfunctions to recover cleanly: this RFC recommends a default timeout
+of 7 days.
+
+[[transaction-submission]]
+==== Submitting the Signed Transaction
+
+Submitting a signed Bitcoin transaction requires a connection to a Bitcoin
+node, direct or indirect. To avoid _strictly requiring_ a connection to a
+Bitcoin node, this RFC proposes logging the signed transaction 5 times on the
+WARN level with a descriptive prefix. Current operator guidance suggests that
+more than 5 WARN level logs within 5 minutes should trigger a notification
+for closer monitoring by operators, ensuring these logs will be noticed. The
+descriptive prefix will further allow for targeted notifications by operators
+so that they can flag these logs for follow-up action.
+
+The suggested structure of the log message is:
+
+```
+2020-02-01T12:12:12.12-0800    WARN   keep-ecdsa   Please broadcast Bitcoin transaction 0010010101000828181124af234gbawed000
+```
+
+The first three components are the standard components emitted by the ECDSA
+client's logging library, while the message itself begins with `Please broadcast
+Bitcoin transaction`, followed by the hexadecimal string representing the signed
+transaction. The signed transaction can be broadcast via manual submission to a
+known node, via API access to a service provider like BlockCypher or
+Blockstream, or via a manual form submission on a page like
+https://live.blockcypher.com/btc/pushtx/ .
+
+If an optional Bitcoin node connection is available (see <<electrs>>), the
+client should attempt to publish the transaction by performing an HTTP POST
+request to path `/tx`. The POST request body should be the signed transaction
+hex, as would be logged in the WARN logs above. If the POST fails, the client
+should log an error and then fall back on the WARN logging above.
+
+=== Implementation Details
+
+There are a few implementation details that are worth calling out; in
+particular, the *pub address derivation, the deposit output script
```

Comment:
You flagged this [earlier in the RFC cycle](https://github.com/keep-network/keep-ecdsa/pull/670#discussion_r561866187) :grin: I think it's beyond the scope of this RFC since it's not about the technical implementation, but I think the RFC can explicitly note that docs should be provided for stakers who want to do this.

Reply:
Oh man, I'm sorry! I was going to remove this comment before submitting the review 😆